packages/osd-test/src/functional_test_runner/integration/failure_hooks.test.js

```diff
@@ -39,11 +39,9 @@ const SCRIPT = resolve(REPO_ROOT, 'scripts/functional_test_runner.js');
 const FAILURE_HOOKS_CONFIG = require.resolve('../fixtures/failure_hooks/config.js');
 
 describe('failure hooks', function () {
-  this.timeout(60 * 1000);
```

Comment:
Same here.

Reply:
<img width="805" alt="Screen Shot 2022-05-26 at 12 34 50 PM" src="https://user-images.githubusercontent.com/1679762/170563737-34e0e35f-b527-46af-80ce-d19920b8be4f.png">

Given that the jest suites for this package run in a couple seconds, I think it's fine.